data/structures.yml

```diff
@@ -902,7 +902,7 @@ US:
   :branch_code_length: 0
   :account_number_length: 17
   :bank_code_format: "\\d{9}"
-  :account_number_format: "\\A_*\\d{1,17}\\z"
+  :account_number_format: "_*\\d{1,17}"
```

Comment:
Might be worth just replicating this fully but I don't get why we've got a * in front

Reply:
Not sure what you mean by reverting this? I removed the `\A` and `\z` as these are now added automatically. I had a look for the context for the `_*` part and im none the wiser - see https://github.com/gocardless/ibandit/commit/d5a4b975b75396468720c07780d5e0eeed66551b for context

Reply:
Ahh, I didn't realise we'd done the whole \\A thing for the US before hand..